src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.NET.Build.Tasks
     public interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
+        string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
```

Comment:
Nothing in `Microsoft.NET.Build.Tasks.dll` is meant to be consumed by callers outside of the .targets files in the dotnet/sdk repo.  Yes, it says `public` and people could try to use the types in this assembly, but we don't package them in any consumable way - i.e. no NuGet package that you can reference and now refer to types in the assembly. If someone tries, and they get broken in the future, they need to fix themselves.

The reason we can't make these types 'internal' and use IVTA right now is because https://github.com/dotnet/sdk/issues/55 isn't fixed.  So the signing of the assemblies gets messed up cross-platform, which leads to complications with IVTA.